libs/wallet-ui/src/components/page/page.spec.tsx

```diff
@@ -0,0 +1,37 @@
+// import { render, screen } from '@testing-library/react'
```

Comment:
These tests are commented out because they cannot be run due to uin toolkit issues. This will be addressed in a follow up PR